maven/plugins/com.liferay.ide.maven.core/plugin.xml

```diff
@@ -43,6 +43,16 @@
       </liferayProjectProvider>
    </extension>
    <extension
+         point="com.liferay.ide.core.liferayProjectProviders">
```

Comment:
The old LiferayMavenProjectProvider  and LiferayMavenModuleProjectProvider they won't have too many class can be shared, the old LiferayMavenProjectProvider contains so many methods that only used for normal liferay plugin project. Its seems better that keep module project provider independence.